nemo/collections/asr/data/audio_to_text.py

```diff
@@ -288,6 +291,151 @@ def __init__(
         )
 
 
+@experimental
+class AudioToCharWithDursDataset(AudioToCharDataset):
+    """
+    Dataset that loads tensors via a json file containing paths to audio
+    files, transcripts, and durations (in seconds). Each new line is a
+    different sample. Example below:
+    {"audio_filepath": "/path/to/audio.wav", "text_filepath":
+    "/path/to/audio.txt", "duration": 23.147}
+    ...
+    {"audio_filepath": "/path/to/audio.wav", "text": "the
+    transcription", "offset": 301.75, "duration": 0.82, "utt":
+    "utterance_id", "ctm_utt": "en_4156", "side": "A"}
+
+    Additionally, user provides path to precomputed durations.
+
+    Args:
+        **kwargs: Passed to AudioToCharDataset constructor.
+        vocab_notation: Vocabulary to be used for text preprocessing.
+        vocab_punct: True if use punctuation for vocab.
+        vocab_spaces: True if use spaces for vocab.
+        vocab_stresses: True if use stresses for vocab.
+        durs_path: String path to pickled durations location.
```

Comment:
I'm doing to need a script of how to create such a durs_path file, or some explanation of what this file looks like.

Reply:
Hello, I need, also, an explanation of durs_path files, what it should look like at least so that I can create a pickle file that runs on the scripts. Thank you for your time.

Reply:
It's pickeld list of (file tag, durations). There will be a separate durs file creation script from pretrained ASR model. It's non-trivial to make and I don't yet have new NeMo ASR ckpts to use, so we need to wait a little while I am working on it.

Reply:
Thank you for your response, I will wait for it then.